ibis/expr/analysis.py

```diff
@@ -1054,6 +1054,30 @@ def predicate(expr):
     return list(lin.traverse(predicate, expr))
 
 
+def find_subqueries(expr):
+    def predicate(expr):
+        op = expr.op()
+
+        if isinstance(op, ops.Join):
+            return [op.left, op.right], None
+        elif isinstance(op, ops.PhysicalTable):
+            return lin.halt, None
+        elif isinstance(op, ops.SetOp):
+            return lin.proceed, op
```

Comment:
```suggestion
```

This isn't necessary because you're handling `TableNode` the same way.

Reply:
Updating.